cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -59,6 +67,12 @@ class InformationMetadata(BaseMetadata):
     updated: datetime = Field(
         description=("Date of the data model update"),
     )
+    contributor: StrListType = Field(
```

Comment:
Should be creator not contributor, since everyone is participating in the creation process.

Reply:
Other two rules profile should e updated accordingly, ie substitute contributor with creator